app/views/api/v2/hostgroups/ansible_roles.json.rabl

```diff
@@ -7,3 +7,7 @@ extends 'api/v2/ansible_roles/show'
 node :inherited do |role|
   @inherited_ansible_roles.include?(role)
 end
+
+node :owned do |role|
```

Comment:
might `directly_assigned` or `own` be better names? :thinking:

Reply:
I thought about these two and I have no idea which one is more straightforward for the users. `directly_assigned` is more descriptive for me and I'd choose this, but it sounds like something more internal / for devs. Thus I've chosen `own`, but with the same form as with `inherited`.

Naming is hard :face_exhaling:

Reply:
I'd go with `directly_assigned`. It might be a slightly more technical name, but I'd say it is warranted here